snippets/rotate.md

```diff
@@ -0,0 +1,27 @@
+### rotate
+
+Rotates the array (in left direction) by the number of shifts.
+
+Given the $shift index, it merges the array values after $shift with the values before $shift.
+
+```php
+function rotate($arr, $shift=1)
+{
+  $resultStarting = array_slice($arr, $shift); //extracting elements starting from $shift position till end
+  $resultEnding = array_slice($arr, 0, $shift); //extracting elements from start till $shift position
+  $result = array_merge($resultStarting, $resultEnding);
```

Comment:
Consider following approach:

```
$numbers = [1, 2, 3, 4];
array_push($numbers, array_shift($numbers));
print_r($numbers);
```
I think this approach is more clear and simple that original approach.

Reply:
@peter279k You could make a PR